### PR TITLE
Add button for switch function control (fireplace/boost)

### PR DESCRIPTION
**Pull request by github-k8n**
Add button for switch function control (fireplace/boost)

### CODE REVIEW

**Comment by kotope** [comment]
@github-k8n you've changed query interval from 5min to 30s. This possibly can lead to more signal conflicts. At least on the master branch the code does respect sending turns and that's why the interval is long to avoid conflicts.

Just want to make sure this is intended and everything works fine even thou decreasing the query interval? I trust you on this one if you say yes.  :-)

**Comment by github-k8n** [comment]
Hmmm... I noticed that the binary sensor for the switch did only update during the next query.
Do you have a bit more details on this? Do you just mean that there is a risk that the regular (every few seconds) messages from the device and the query might interfere?

it is only about the sensor showing the right information and because we are sending it  and not receiving from another device thus that will not be immediately reflected.

I guess I just have to update the sensor once I trigger the switch, that way we avoid the delay. 

Long story short, I might add some other changes, if you want you can merge this with the query interval moved back to 5 min..  or you can wait until i have some update (should be able to address this sometime later this week..)

**Comment by github-k8n** [comment]
Ok, changed it back to 5 min just to be on the safe side.

**Comment by kotope** [comment]
Thanks @github-k8n ! Great job 👍
